{% if cookiecutter.library_prefix %}{{ cookiecutter.library_prefix | capitalize }}_CircuitPython{% else %}CircuitPython_Org{% endif %}_{{ cookiecutter.library_name|replace(" ", "_")}}/{% if cookiecutter.target_bundle == 'Community' %}CODE_OF_CONDUCT.md{% endif %}

```diff
@@ -0,0 +1,139 @@
+<!--
+SPDX-FileCopyrightText: 2014 Coraline Ada Ehmke
+SPDX-FileCopyrightText: 2019-2021 Kattni Rembor
+
+SPDX-License-Identifier: CC-BY-4.0
+-->
+# CircuitPython Community Code of Conduct
+
+## Our Pledge
+
+In the interest of fostering an open and welcoming environment, we as
+contributors and leaders pledge to making participation in our project and
+our community a harassment-free experience for everyone, regardless of age, body
+size, disability, ethnicity, gender identity and expression, level or type of
+experience, education, socio-economic status, nationality, personal appearance,
+race, religion, or sexual identity and orientation.
+
+## Our Standards
+
+We are committed to providing a friendly, safe and welcoming environment for
+all.
+
+Examples of behavior that contributes to creating a positive environment
+include:
+
+* Be kind and courteous to others
```

Comment:
Would it make sense to make this use the same tense as the other points, that is, "Being kind and courteous to others"?

Reply:
Yes it probably would make sense.